evalml/tests/conftest.py

```diff
@@ -158,6 +158,19 @@ def X_y_categorical_classification():
     return X, y
 
 
+@pytest.fixture()
```

Comment:
Moved this here, since LSA and TextFeaturizer both use it :)

Reply:
Good call!